articleList.scss

```diff
@@ -0,0 +1,36 @@
+@import "../variables";
+
+.insights-articleList {
+    @include transform(translateY(10px));
+    opacity: 100;
+  
+    .articleList-article p {
+      color: #000;
+      font-weight: bold;
+      font-size: 22px;
+      line-height: 22px;
+      
+      a {
+        color: $color--grey6;  
+        text-transform: underline;
+      }
+    }
+
+    .articleList-article {
+        border: 1px solid $color--insights-articlelist--border;
+        border-left: 0;
+        flex: 1 0 30%;
+        max-width: 33.3%;
+        box-sizing: content-box;
```

Comment:
In my opinion, box-sizing:border-box is more predictable, however i don't know context of using content-box in this project